src/ci/docker/dist-x86_64-illumos/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04
```

Comment:
The docker image needs to be moved to `src/ci/docker/host-x86_64/dist-x86_64-illumos`, since #73650 was merged before this.